### PR TITLE
Project 1: Cloning A New York Times Article page.

### PR DESCRIPTION
**Pull request by crappypapa**
I made a clone of a **New York Times article page** by learning and applying the following concepts to the project:

-  Embedding images
- The box model
- Flexbox
- Positioning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://crappypapa.github.io/New_York_Times_Article/)
+
```
Comment (whiz25) [inline README.md:15]: - Add instructions for cloning, starting the app locally. Refer to [this](https://github.com/microverseinc/readme-template) template

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice work on this PR. You app looks good. You covered most requirements for this PR. 
We however have to fix a few things before we can merge this PR.
- [x] Fix the footer. It's not similar to the given [design](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)
This is how your footer looks like:
Screenshot from 2021-02-17 16-16-52
- [x] The `Space & Cosmos` title of the navbar should be a link. Please make it a link.
- [x] Also make the New York Times a link.
This is for the original page:
Screenshot from 2021-02-17 16-20-05
#### Your footer lacks the sixth (Subscribe) column. Also add some margin bottom to the `<div class="footer-big">` div to give room to the elements
- [x] You missed to include the `Suggested newsletters for you`, `More in Space and Astronomy` and `Editors’ Picks` sections. 

This is what you have:
Screenshot from 2021-02-17 16-28-32

This is what's in the original design
Screenshot from 2021-02-17 16-19-59
#### Make sure your page looks close to the original as possible.

- [x] Be consistent in your commit messages. `fix base footer ` should be capitalized. So the accepted format is `Fix base footer` and `Added Linters, added index and style files` should be `Add linters, add index and style files`. Use present tense not past.
- [x] Make changes and send for another review.

### Martin Afani

File: README.md (hunk 1 above)
Comment (crappypapa) [reply README.md:15]: I have implemented the requested changes.

**Review by MarilenaRoque — APPROVED** [review]
## Hello Team! :rocket:

Congratulations on completing this project.
Excellent job implementing all the requested changes :tada:

# Your Project is Approved! 🥇 

[ FOR FUTURE PROJECTS] Make sure to be consistent with the capitalized letters on your PR. It will make your repository look more professional. 👍🏻 
Keep going with the hard work! 💪🏻 
Happy Coding! 💻 

**_Marilena Roque
Full-Stack Developer
Code Reviewer at Microverse Inc.
[Portfolio](https://mariroqueproject.com/) | [Linkedin](https://www.linkedin.com/in/roquemarilena/) 
marilenas@id.uf.br_**

**Pull request merged**